### ISSUE TITLE
Feature request: set extends in source

### ISSUE THREAD

**Issue by felipecrs** [nvuillam/npm-groovy-lint#138]
I have both Groovy and Jenkinsfiles in my repo, and I would like to lint each against a different set of rules. Since I just have a couple of Jenkinsfiles, it would be very welcome if I could set this at the beginning of the file:

```groovy
/* groovylint-extends recommended-jenkinsfile */
```

This is probably another fix for https://github.com/nvuillam/npm-groovy-lint/issues/79.

**Comment by nvuillam** [comment]
CodeNarc is called once with a set of rules, applicable to all found files... I'm sorry, the architecture does not allow that, I don't have anything else to propose than:
- making 2 npm-groovy-lint calls
- Have a single .groovylintrc.json , and add `groovylint-disable` comments in Jenkinsfile to deactivate the rules you don't want to use

**Comment by felipecrs** [comment]
> making 2 npm-groovy-lint calls

Actually, I'm using the VSCode extension, so it would not help in my case. =/
